From 2201c4837ce65dcc2c5d1fd4e79044905423119b Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Sun, 21 Feb 2016 13:55:27 +0200
Subject: [PATCH 141/538] i2c: mv64xxx: enabled marvell i2c driver for mvebu
 platforms

This enables using the driver also in the arm64 platforms that
dont use PLAT_ORION

Change-Id: Ic32b6b1dce84226e4d3e1da59e5d66d222e2d338
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
---
 drivers/i2c/busses/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/i2c/busses/Kconfig b/drivers/i2c/busses/Kconfig
index 7b0aa82..437c58e 100644
--- a/drivers/i2c/busses/Kconfig
+++ b/drivers/i2c/busses/Kconfig
@@ -662,7 +662,7 @@ config I2C_MT65XX
 
 config I2C_MV64XXX
 	tristate "Marvell mv64xxx I2C Controller"
-	depends on MV64X60 || PLAT_ORION || ARCH_SUNXI
+	depends on MV64X60 || PLAT_ORION || ARCH_MVEBU || ARCH_SUNXI
 	help
 	  If you say yes to this option, support will be included for the
 	  built-in I2C interface on the Marvell 64xxx line of host bridges.
-- 
1.9.1

